pom.xml

```
@@ -0,0 +1,545 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>io.github.zentol.flink</groupId>
+		<artifactId>flink-connector-parent</artifactId>
+		<version>1.0</version>
+	</parent>
+
+	<groupId>org.apache.flink</groupId>
+	<artifactId>flink-connector-hbase-parent</artifactId>
+	<version>1.0-SNAPSHOT</version>
+
+	<name>Flink : Connectors : HBase Parent</name>
+	<packaging>pom</packaging>
+	<url>https://flink.apache.org</url>
+	<inceptionYear>2022</inceptionYear>
+
+	<licenses>
+		<license>
+			<name>The Apache Software License, Version 2.0</name>
+			<url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+			<distribution>repo</distribution>
+		</license>
+	</licenses>
+
+	<scm>
+		<url>https://github.com/apache/flink-connector-hbase</url>
+		<connection>git@github.com:apache/flink-connector-hbase.git</connection>
+		<developerConnection>scm:git:https://gitbox.apache.org/repos/asf/flink-connector-hbase.git</developerConnection>
+	</scm>
+
+	<properties>
+		<scala.binary.version>2.12</scala.binary.version>
+
+		<flink.version>1.16.0</flink.version>
+		<flink.shaded.version>16.0</flink.shaded.version>
```

```suggestion
		<flink.shaded.version>15.0</flink.shaded.version>
```
This is the actual version that 1.16.0 depends on.